### PR TITLE
Fix README typo

### PR DESCRIPTION
**Pull request by laurentboileau**
## Description

There was a typo in the `trikot.analytics` module description.

## Motivation and Context

Details matter. 😉

### CODE REVIEW

**Review by remi — APPROVED** [review]

**Comment by marcantoinefortier** [comment]
https://github.com/mirego/trikot/pull/12#discussion_r789798613 😅

**Review by marcantoinefortier — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Trikot is a framework that helps building kotlin multiplatform apps. iOS, androi
 | [trikot.datasources](./trikot-datasources)                       | Cascading data access layers abstraction.                           |
 | [trikot.http](./trikot-http)                                     | Multiplatform http client with native platform implementations.     |
 | [trikot.graphql](./trikot-graphql)                               | GraphQL query client built over trikot.http and trikot.datasources. |
-| [trikot.analytics](./trikot-analytics)                           | AAndroid and iOS analytics providers.                               |
+| [trikot.analytics](./trikot-analytics)                           | Android and iOS analytics providers.                               |
```
Comment (remi) [inline README.md:30]: Just need to run Prettier 🙂 
```suggestion
| [trikot.analytics](./trikot-analytics)                           | Android and iOS analytics providers.                                |
```

Comment (laurentboileau) [reply README.md:30]: On it.

**Comment by npresseault** [comment]
oups!

**Pull request merged**